.github/workflows/ci.yml

```
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1.39.0 # MSRV
+          - 1.40.0
```

```suggestion
          - 1.40.0 #MSRV
```

Let's leave the comment so that we don't miss the purpose.